app/src/main/java/com/sun/basic_japanese/audiolessons/adapter/AudioLessonsRecyclerAdapter.kt

```diff
@@ -21,16 +22,18 @@ class AudioLessonsRecyclerAdapter(
     }
 
     override fun onBindViewHolder(viewHolder: ViewHolder, position: Int) {
-        viewHolder.bindData(lessonItems[position])
+        viewHolder.bindData(lessons[position].first, lessons[position].second)
```

Comment:
```suggestion
        viewHolder.bindData(lessons[position])
```
Gửi cả Pair vào cho ViewHolder tự xử lý